spec/fixtures/sample.js

```diff
@@ -10,4 +10,20 @@ var quicksort = function () {
   };
 
   return sort(Array.apply(this, arguments));
-};
\ No newline at end of file
+};
+
+foo({
+    atIndentTwo: true
```

Comment:
Some of this indentation looks incorrect to me. For example, take this code from the [node.js docs](https://nodejs.org/api/http.html#http_class_http_agent):

```js
http.get({
  hostname: 'localhost',
  port: 80,
  path: '/',
  agent: false  // create a new agent just for this one request
}, (res) => {
  // Do stuff with response
})
```

The keys in object literal, and the body of the function should be indented only one level.